fades/pipmanager.py

```diff
@@ -108,3 +108,10 @@ def _brute_force_install_pip(self):
         python_exe = os.path.join(self.env_bin_path, "python")
         helpers.logged_exec([python_exe, self.pip_installer_fname, '-I'])
         self.pip_installed = True
+
+    def freeze(self, filepath):
+        """Dump venv contents to the indicated filepath."""
+        logger.debug("running freeze to store in %r", filepath)
+        stdout = helpers.logged_exec([self.pip_exe, "freeze", "--all", "--local"])
+        with open(filepath, "wt", encoding='utf8') as fh:
```

Comment:
why are you adding `t` here? That's just the default

Reply:
Explicit better than implicit :).

Now seriously, I always write 't' or 'b' there, as the difference is getting bytes or unicode chars, which may be critical in the following handling of the file handler.